### ISSUE TITLE
[policy] rate limiting errors on pull requests

### ISSUE THREAD

**Issue by JustinBeckwith** [googleapis/repo-automation-bots#1690]
Looking in the logs here:
https://pantheon.corp.google.com/logs/query;cursorTimestamp=2021-04-27T09:01:08.628Z;query=resource.type%3D%22cloud_function%22%0Aseverity%3DERROR%0Atimestamp%3D%222021-04-27T09:01:08.171Z%22%0AinsertId%3D%22000000-e75da81d-55d2-4fe2-b84e-7dc8cb3bab68%22;timeRange=P1D?project=repo-automation-bots

There are a slew of rate limiting errors directly related to the new `SECURITY.md` PR that landed yesterday.  Given the generous API limits with GitHub App tokens, I was really surprised to see us getting limited.  This;

```
{
  "insertId": "000000-e75da81d-55d2-4fe2-b84e-7dc8cb3bab68",
  "jsonPayload": {
    "type": "Error",
    "documentation_url": "https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting",
    "level": 50,
    "request": {
      "request": {},
      "url": "https://api.github.com/search/issues?q=repo%3Agoogleapis%2Fgoogle-cloud-php-service-directory%20%22chore%3A%20add%20a%20Code%20of%20Conduct%22%20in%3Atitle%20is%3Aopen",
      "method": "GET",
      "headers": {
        "authorization": "token [REDACTED]",
        "user-agent": "octokit-rest.js/18.3.5 octokit-rest.js/18.3.5 octokit-core.js/3.3.1 Node.js/12.22.0 (linux; x64)",
        "accept": "application/vnd.github.v3+json"
      }
    },
    "trigger": {
      "github_delivery_guid": "c3dd5fe3-0186-4503-aeba-a9729023fc12",
      "trigger_type": "Cloud Task"
    },
    "stack": "HttpError: API rate limit exceeded for installation ID 15630733.\n    at /workspace/node_modules/@octokit/request/dist-node/index.js:66:23\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)\n    at async sendRequestWithRetries (/workspace/node_modules/@octokit/auth-app/dist-node/index.js:445:12)\n    at async addCodeOfConduct (/workspace/build/src/changer.js:60:17)\n    at async Object.submitFixes (/workspace/build/src/changer.js:140:9)\n    at async /workspace/build/src/bot.js:52:13\n    at async Promise.all (index 0)\n    at async /workspace/node_modules/gcf-utils/build/src/gcf-utils.js:229:21",
    "name": "HttpError",
    "message": "API rate limit exceeded for installation ID 15630733.",
    "headers": {
      "x-ratelimit-reset": "1619514073",
      "content-security-policy": "default-src 'none'",
      "access-control-expose-headers": "ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, Deprecation, Sunset",
      "content-type": "application/json; charset=utf-8",
      "x-ratelimit-used": "100",
      "x-github-request-id": "B541:3667:3DE70EA:6D77A61:6087D2D4",
      "vary": "Accept-Encoding, Accept, X-Requested-With",
      "content-encoding": "gzip",
      "strict-transport-security": "max-age=31536000; includeSubdomains; preload",
      "x-ratelimit-limit": "30",
      "x-ratelimit-remaining": "0",
      "x-github-media-type": "github.v3; format=json",
      "transfer-encoding": "chunked",
      "connection": "close",
      "server": "GitHub.com",
      "date": "Tue, 27 Apr 2021 09:01:08 GMT",
      "referrer-policy": "origin-when-cross-origin, strict-origin-when-cross-origin",
      "x-xss-protection": "0",
      "x-frame-options": "deny",
      "access-control-allow-origin": "*",
      "x-content-type-options": "nosniff"
    },
    "status": 403
  },
  "resource": {
    "type": "cloud_function",
    "labels": {
      "project_id": "repo-automation-bots",
      "region": "us-central1",
      "function_name": "policy"
    }
  },
  "timestamp": "2021-04-27T09:01:08.171Z",
  "severity": "ERROR",
  "labels": {
    "execution_id": "ulbzczuuhvkh"
  },
  "logName": "projects/repo-automation-bots/logs/cloudfunctions.googleapis.com%2Fcloud-functions",
  "trace": "projects/repo-automation-bots/traces/f338e60f39748cb44614dae886a43df3",
  "receiveTimestamp": "2021-04-27T09:01:17.953574541Z"
}
```

The retries are purposefully not working here because we swallow and log errors related to autofixing.

This may be normal and fine, but I want to park this here so I can investigate if I'm doing something wrong

**Comment by JustinBeckwith** [comment]
lol https://docs.github.com/en/rest/reference/search#:~:text=The%20Search%20API%20has%20a,to%2010%20requests%20per%20minute.

**Comment by JustinBeckwith** [comment]
Fixed in #1697
